libc/functions/t/tolower.part-impl.md

```diff
@@ -0,0 +1,42 @@
+# Synopsis
+
+`#include <ctype.h>`
+
+`int tolower(int c);`
+
+## Description
+
+The `tolower()` function has as a domain a type int, the value of which is representable as a `unsigned char` or the
+value of `EOF`.
+
+If the argument has any other value, the behavior is undefined.
+
+If the argument of `tolower()` represents an uppercase letter, and there exists a corresponding lowercase letter as
+defined by character type information in the current locale respectively the result shall be the corresponding lowercase
+letter.
+
+All other arguments in the domain are returned unchanged.
+
+## Return value
+
+Upon successful completion, the `tolower()` function shall return the lowercase letter corresponding to the argument
+passed.
+
+Otherwise, they shall return the argument unchanged.
+
```

Comment:
```suggestion
Upon successful completion, the `tolower()` function shall return the lowercase letter corresponding to the argument
passed, otherwise, they shall return the argument unchanged.

```